rho/facts.py

```diff
@@ -193,6 +193,8 @@ def new_fact(fact_name, description, is_default=None,
 new_fact('jboss.eap.summary', 'Summary of JBoss EAP information',
          is_default=True, categories=[JBOSS_FACTS])
 new_fact('jboss.activemq-ver', 'ActiveMQ version', is_default=False)
+new_fact('jboss.brms.summary', 'Summary of BRMS information',
```

Comment:
Should this be in the JBOSS_FACTS category? like EAP and Fuse?

Reply:
Yep, thanks for catching that.